Keyboards/KeyboardsBase/KeyboardViewController.swift

```diff
@@ -380,7 +385,6 @@ class KeyboardViewController: UIInputViewController {
   /// - Parameters
   ///   - keyPressed: the auto action button that was executed.
   func executeAutoAction(keyPressed: UIButton) {
-    autoActionState = .suggest
```

Comment:
I think that this was because it was canceling out `.invalid` and `.alreadyPlural`, so we'd never see those states :) Let's keep this change in mind though.